library/sketches-library/src/test/java/uk/gov/gchq/gaffer/sketches/datasketches/cardinality/function/IterableToHllSketchTest.java

```diff
@@ -112,6 +112,24 @@ public void shouldCorrectlyCreateFromAnotherHllSketch() {
         assertThat(result.getLgConfigK()).isEqualTo(5);
     }
 
+    @Test
+    public void shouldCorrectlyCopyAnotherHllSketch() {
+        // Given
+        final HllSketch anotherSketch = new HllSketch();
+        anotherSketch.update("second");
+        anotherSketch.update("third");
+
+        IterableToHllSketch iterableToHllSketch = new IterableToHllSketch(anotherSketch);
+        final List<Object> input = Arrays.asList("one", "one", "two", "two", "three");
```

Comment:
Pretty minor but could be `List<String>` instead?

Reply:
When I do that `iterableToHllSketch.apply` is unhappy as it specifies `Iterable<Object>`.

Reply:
Fair makes sense